### PR TITLE
Adding css styling to the app

### PR DESCRIPTION
**Pull request by pranathip**
# Summary
Added some css styling to our game

# Images

### CODE REVIEW

**Comment by pranathip** [comment]
Current dependencies on/for this PR:
* main
  * **PR #21** <a href="https://app.stg.graphite.dev/github/pr/pranathip/pranathi-2048-react-app/21" target="_blank">Graphite</a> 
    * **PR #22** <a href="https://app.stg.graphite.dev/github/pr/pranathip/pranathi-2048-react-app/22" target="_blank">Graphite</a> 
      * **PR #23** <a href="https://app.stg.graphite.dev/github/pr/pranathip/pranathi-2048-react-app/23" target="_blank">Graphite</a> 
        * **PR #24** <a href="https://app.stg.graphite.dev/github/pr/pranathip/pranathi-2048-react-app/24" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.stg.graphite.dev/github/pr/pranathip/pranathi-2048-react-app/24?utm_source=stack-comment).

**Comment by pranathip** [comment]
This is an issue level comment

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,78 @@
...
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.row {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-gap: 8px;
```
Comment (pranathip) [inline src/App.css:21]: Does this gap look ok?

**Review by pranasaurusrex — APPROVED** [review]

**Comment by pranathip** [comment]
@pranathip merged this pull request with [Graphite](https://app.graphite.dev/github/pr/Avocadoo-Corp/pranathi-2048-react-app/24).

**Pull request merged**